### ISSUE TITLE
 sqlalchemy.exc.InvalidRequestError: Vws friend stopped recording data

### ISSUE THREAD

**Issue by daniel-w00** [tillsteinbach/VWsFriend#324]
Vws friend had an error and stopped recording data after trip ended and charger connected.
After restarting the container everything worked fine.

I don't know if this has something to do with it, but this was the first trip after the 2.4 update.
Log:
```
2022-09-02T13:01:44+0000:INFO:vwsfriend_base:Updating data from WeConnect

2022-09-02T13:01:46+0000:INFO:trip_agent:Vehicle WVWZZZE1Z*** ended a trip (car was connected to charger)

2022-09-02T13:01:46.831238452Z Traceback (most recent call last):

2022-09-02T13:01:46.831304284Z   File "/opt/venv/bin/vwsfriend", line 8, in <module>   sys.exit(main())

2022-09-02T13:01:46.842102712Z   File "/opt/venv/lib/python3.10/site-packages/vwsfriend/vwsfriend_base.py", line 493, in main
    weConnect.update(updateCapabilities=True, updatePictures=True, force=True)

2022-09-02T13:01:46.849257559Z   File "/opt/venv/lib/python3.10/site-packages/weconnect/weconnect.py", line 183, in update
     self.updateComplete()

2022-09-02T13:01:46.851045127Z   File "/opt/venv/lib/python3.10/site-packages/weconnect/addressable.py", line 519, in  
    updateComplete    child.updateComplete()

2022-09-02T13:01:46.853010841Z   File "/opt/venv/lib/python3.10/site-packages/weconnect/addressable.py", line 519, in updateComplete  child.updateComplete()

2022-09-02T13:01:46.853859867Z   File "/opt/venv/lib/python3.10/site-packages/weconnect/addressable.py", line 519, in updateComplete  child.updateComplete()

2022-09-02T13:01:46.854587061Z   [Previous line repeated 3 more times]

2022-09-02T13:01:46.854616134Z   File "/opt/venv/lib/python3.10/site-packages/weconnect/addressable.py", line 97, in updateComplete  observer(element=self, flags=self.onCompleteNotifyFlags)

2022-09-02T13:01:46.855075424Z   File "/opt/venv/lib/python3.10/site-packages/vwsfriend/agents/charge_agent.py", line 200, in __onPlugConnectionStateChange  self.session.refresh(self.chargingSession)

2022-09-02T13:01:46.860994418Z   File "/opt/venv/lib/python3.10/site-packages/sqlalchemy/orm/session.py", line 2348, in refresh

2022-09-02T13:01:46.865687651Z     raise sa_exc.InvalidRequestError(

2022-09-02T13:01:46.865847463Z sqlalchemy.exc.InvalidRequestError: Could not refresh instance '<ChargingSession at 0xb425f2f8>
```

**Comment by tillsteinbach** [comment]
Hmm, I have not yet succeeded in finding out what exactly causes this problem.

**Comment by tillsteinbach** [comment]
I have a small hope that this will be fixed with the update of sqlalchemy to 2.0.0 that is currently in testing.
